sdk/containerorchestratorruntime/Azure.ResourceManager.ContainerOrchestratorRuntime/src/Generated/Models/ProvisioningState.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.ContainerOrchestratorRuntime.Models
+{
+    /// <summary> The status of the current operation. </summary>
+    public readonly partial struct ProvisioningState : IEquatable<ProvisioningState>
```

Comment:
`ProvisioningState` is a very common word across various RPs. Better to add certain prefix, like `ContainerOrchestrator`

Reply:
Changed to `ContainerOrchestratorProvisioningState`